gtt.go

```diff
@@ -0,0 +1,204 @@
+package kiteconnect
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// GTTType represents the available GTT order types.
+type GTTType string
+
+const (
+	// GTTTypeSingle is used to monitor a single trigger value
+	GTTTypeSingle GTTType = "single"
+	// GTTTypeOCO is used to monitor two trigger values
+	// where executing one cancels the other.
+	GTTTypeOCO GTTType = "two-leg"
+)
+
+// GTTs represents a list of GTT orders.
+type GTTs []GTT
+
+// GTTMeta contains information about the rejection reason
+// received after GTT order was triggered.
+type GTTMeta struct {
+	RejectionReason string `json:"rejection_reason"`
+}
+
+// GTTCondition represents the condition inside a GTT order.
+type GTTCondition struct {
+	Exchange      string    `json:"exchange"`
+	Tradingsymbol string    `json:"tradingsymbol"`
+	LastPrice     float64   `json:"last_price"`
+	TriggerValues []float64 `json:"trigger_values"`
+}
+
+// GTT represents a single GTT order.
+type GTT struct {
+	ID            int          `json:"id"`
+	UserID        string       `json:"user_id"`
+	ParentTrigger interface{}  `json:"parent_trigger"`
+	Type          GTTType      `json:"type" url:""`
+	CreatedAt     string       `json:"created_at"`
+	UpdatedAt     string       `json:"updated_at"`
+	ExpiresAt     string       `json:"expires_at"`
+	Status        string       `json:"status"`
+	Condition     GTTCondition `json:"condition"`
+	Orders        []Order      `json:"orders"`
+	Meta          GTTMeta      `json:"meta"`
+}
+
+// Trigger is an abstraction over multiple GTT types.
+type Trigger interface {
+	TriggerValues() []float64
+	LimitPrices() []float64
+	Quantities() []float64
+}
+
+type TriggerParams struct {
+	TriggerValue float64
+	LimitPrice   float64
+	Quantity     float64
```

Comment:
Make it as integer.

Reply:
orders.go also is using float64

https://github.com/zerodhatech/gokiteconnect/blob/master/orders.go#L36